WordPress/gutenberg
# Page List : Add border and spacing support

## What?

Adding block & spacing support for Page list block.

Part of https://github.com/WordPress/gutenberg/issues/43247 and  https://github.com/WordPress/gutenberg/issues/43243


## Why?

Page list block is missing border & spacing  support

## How?

Adds the border & spacing  support via block.json


## Testing Instructions

- Go to Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
- Make sure that `Page List `block's border is configurable via Global Styles
- Verify that Global Styles are applied correctly in the editor and frontend
- Edit template/page, Add `Page List` block and apply the border styles & spacing (padding, margin)styles
- Verify that block styles take precedence over global styles
- Verify that block borders & spacing display correctly in both the editor and frontend



## Screenshots or screencast 

In Backend:
![pagelistbackend](https://github.com/user-attachments/assets/686481e4-4322-4c12-875f-89ae4ed465e5)

In Frontend:
![pagelistfrontend](https://github.com/user-attachments/assets/65e24feb-a465-4da8-b857-958e659cbb85)
